# scRNA-scATAC-related-knowledge
Recording some informative papers, blogs, tutorials and threads (ordered by its issue number (latest to oldest))

## Theoretical knowledge
### Papers
[**The triumphs and limitations of computational methods for scRNA-seq**](https://www.nature.com/articles/s41592-021-01171-x)  

### Blogs, Youtube or Other sources
[**流型学习(mainfold learning)**](https://www.zhihu.com/question/24015486).   
[**Ergodic process**](https://www.youtube.com/watch?v=VCb2AMN87cg)


## Practical knowledge
### Tutorials
[**Performing integration on datasets normalized with SCTransform**](https://satijalab.org/seurat/articles/integration_introduction.html#performing-integration-on-datasets-normalized-with-sctransform-1)

[**FindMarkers source code process analysis**](https://programmer.ink/think/shengxin-findmarkers-source-code-process-analysis-default-parameter.html)

[**AddModuleScore**](https://www.waltermuskovic.com/2021/04/15/seurat-s-addmodulescore-function/)

[**Tips for large dataset**](Tips for integrating large datasets). 

### Threads
[**Elaborate FindMarkers() and AverageExpression() for Seurat v4#4210**](https://github.com/satijalab/seurat/discussions/4210)


[**Differential expression testing post-integration#4000**](https://github.com/satijalab/seurat/discussions/4000)


[**Meaning of counts, data and scale.data slots in SCTransformed data #3711**](https://github.com/satijalab/seurat/issues/3711)

[**Using "latent.vars" in FindMarkers function is not helpful#3369**](https://github.com/satijalab/seurat/discussions/3369)

[**subset(downsample= X) #3033**](https://github.com/satijalab/seurat/issues/3033)

[**Adjusting DoHeatMap color scale #1531**](https://github.com/satijalab/seurat/issues/1531).  

[**Randomly downsample seurat object #3108**](https://github.com/satijalab/seurat/issues/3108).  

[**What are the differences between vars.to.regress and latent.vars? Should I use both for a variable? #4259**](https://github.com/satijalab/seurat/discussions/4259)

[**Integration workflow doubts #3665**](https://github.com/satijalab/seurat/issues/3665)  

[**Building a Custom T2T reference for Cell Ranger ARC**](https://kb.10xgenomics.com/hc/en-us/articles/29207065679501-Building-a-Custom-T2T-reference-for-Cell-Ranger-ARC)  

[**Build Notes for Reference Packages**](https://www.10xgenomics.com/cn/support/software/cell-ranger/downloads/cr-ref-build-steps)  

[**reference release note**](https://www.10xgenomics.com/cn/support/software/cell-ranger/latest/release-notes/cr-reference-release-notes)  

[**Download Center**](https://www.10xgenomics.com/support/software/cell-ranger-arc/downloads)  

### OCM analysis
[**CellRanger multi**](https://www.10xgenomics.com/support/software/cell-ranger/latest/analysis/running-pipelines/cr-3p-multi#ocm)   
[**Calling Cell Barcode**](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.10xgenomics.com%2Fsupport%2Fsoftware%2Fcell-ranger%2Flatest%2Falgorithms-overview%2Fcr-gex-algorithm%23cell_calling&data=05%7C02%7Ckf2799%40cumc.columbia.edu%7C2825bea16ff5434f1dc008de43260d49%7Cb0002a9b0017404d97dc3d3bab09be81%7C0%7C0%7C639022027911293631%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=g5c2pUTz5ciMcRL65Aki2Fu%2FmPpsJfpJGczrD7mlvW0%3D&reserved=0) 

[**Cell Ranger's Gene Expression Algorithm**](https://www.10xgenomics.com/support/software/cell-ranger/latest/algorithms-overview/cr-gex-algorithm#cell_calling)  

[**ambient RNA**](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fkb.10xgenomics.com%2Fs%2Farticle%2F12123695108493-What-can-I-do-if-my-single-cell-nuclei-suspension-contains-a-high-amount-of-debris&data=05%7C02%7Ckf2799%40cumc.columbia.edu%7C2825bea16ff5434f1dc008de43260d49%7Cb0002a9b0017404d97dc3d3bab09be81%7C0%7C0%7C639022027911277122%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=cenlV9CAlnLM0gzSVCgIljz%2B22UDQgVDhOwmhp%2FTMM4%3D&reserved=0)  
